src/services/google-analytics.ts

```diff
@@ -0,0 +1,75 @@
+import { VerificationFragment } from "@govtechsg/oa-verify";
+import { OpenAttestationDocument } from "@govtechsg/open-attestation";
+import get from "lodash.get";
+import { useEffect } from "react";
+import ReactGA from "react-ga4";
+
+const GTAG_ID = process.env.REACT_APP_GTAG_ID;
+
+export const useGoogleAnalytics = (): void => {
+  /* Initialise Google Analytics 4 if GTAG_ID is provided */
+  useEffect(() => {
+    try {
+      if (GTAG_ID?.startsWith("G-")) {
+        ReactGA.initialize(GTAG_ID);
+        ReactGA.send("pageview");
+      }
+    } catch (error) {
+      console.error(error);
+    }
+  }, []);
+};
+
+enum HEALTHCERT_TYPE {
+  PDT = "PDT",
+  VAC = "VAC",
+}
+
+const isVac = (data: OpenAttestationDocument): boolean => get(data, "name") === "VaccinationHealthCert";
+const isPDT = (data: OpenAttestationDocument): boolean =>
+  get(data, "name") === "HealthCert" || get(data, "version") === "pdt-healthcert-v2.0";
```

Comment:
@eugbyte Laurent was asking if we can just use `?.` or `.` operator instead of `lodash.get`?

```suggestion
const isVac = (data: OpenAttestationDocument): boolean => data.name === "VaccinationHealthCert";
const isPDT = (data: OpenAttestationDocument): boolean =>
  data.name === "HealthCert" || data.version === "pdt-healthcert-v2.0";
```

Reply:
ok